### ISSUE TITLE
Allow defining `index.html` per output target.

### ISSUE THREAD

**Issue by tricki** [ionic-team/stencil#1884]
**Stencil version:**

```
@stencil/core@1.3.2
```

**I'm submitting a:**

[ ] bug report
[x] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
The same `index.html` will be used for all `www` output targets. Only the name of the generated file can be configured using the `indexHtml` config property.

**Expected behavior:**
Add a setting to use a different HTML file from the `src` directory as the source of the generated `index.html`.

**Steps to reproduce:**
n/a

**Other information:**
We need this because we deploy our project as a Cordova app as well as on the web in a subfolder. Because of this in the web output target the paths to the CSS etc. have to have this subfolder prepended, but not in the output target used to build the app.

**Comment by joewoodhouse** [comment]
@tricki did you find a workaround for this?

**Comment by tricki** [comment]
No, I just replace it manually before each build. But you could write a small node script to replace it automatically.

**Comment by GregBahrey** [comment]
Any updates on this, figured I'd ask since it's been a few years.
